components/gc-follow-us/include.html

```diff
@@ -1,9 +1,17 @@
 <section class="provisional gc-followus">
 	<h2>
-		{%- if i18nText-lang == "fr" -%}
-			Dans les médias sociaux
+		{%- if page.followUsLayout -%}
```

Comment:
This should be passed through an include parameter not a page parameter. Take a look at the provisional alert include file.

ex: `{{ include.siteroot }}`